be/src/olap/rowset/segment_v2/segment_writer.cpp

```diff
@@ -242,6 +236,7 @@ Status SegmentWriter::_create_column_writer(uint32_t cid, const TabletColumn& co
     CHECK_FIELD_TYPE(OBJECT, "object")
     CHECK_FIELD_TYPE(HLL, "hll")
     CHECK_FIELD_TYPE(QUANTILE_STATE, "quantile_state")
+    CHECK_FIELD_TYPE(VARIANT, "variant")
```

Comment:
set bf property in `CHECK_FIELD_TYPE` and rename `CHECK_FIELD_TYPE`

Reply:
done